examples/python.KStar/bbkstar.py

```diff
@@ -36,55 +36,60 @@
 
 # how should we compute energies of molecules?
 # (give the complex conf space to the ecalc since it knows about all the templates and degrees of freedom)
-parallelism = osprey.Parallelism(cpuCores=4)
+parallelism = osprey.Parallelism(cpuCores=20)
```

Comment:
Most people probably won't have 20 CPU cores on their machines while exploring Python examples. Consider reverting back to 4?